README.md

```diff
@@ -2,10 +2,18 @@
 
 <img align="right" src="docs/images/castle-logo.png">
 
+[![Join the chat at https://gitter.im/castleproject/all](https://badges.gitter.im/castleproject/all.svg)](https://gitter.im/castleproject/all?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![NuGet](https://img.shields.io/nuget/v/Castle.Core.svg)](https://www.nuget.org/packages/Castle.Core/)
```

Comment:
AppVeyor token: 49a6i0ydiku56r5b

Maybe we should we put the NuGet badge under the Releases heading?